scripts/run-app.sh

```diff
@@ -1,55 +1,56 @@
 #!/usr/bin/env sh
 
 log() {
-  local message=$1
-  local iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
-  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$1"'"}'
+  message=$1
+  iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
+  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$message"'"}'
 }
 
 export_secrets_from_dir() {
-  local secrets_dir=$1
+  for filename in "$SECRETS_DIR"/*; do
+    filename_without_path=${filename##*/}
+    filename_uppercased=$(echo "$filename_without_path" | tr '[:lower:]' '[:upper:]');
+    filename_with_underscores="$(echo "$filename_uppercased" | tr '-' '_')"
 
-  for filename in $SECRETS_DIR/*; do
-    local filename_without_path=${filename##*/}
-    local filename_uppercased=$(echo $filename_without_path | tr 'a-z' 'A-Z');
-    local filename_with_underscores=${filename_uppercased//-/_}
-
-    local secret_name="SECRET_$filename_with_underscores";
+    secret_name="SECRET_$filename_with_underscores";
 
     log "Exporting secret '$filename_without_path' as '$secret_name'"
 
-    local secret=$(cat $filename);
+    secret=$(cat "$filename");
 
     export "$secret_name=$secret"
   done
 }
 
 start_app() {
   # npm provides binaries on PATH in scripts, so we have to do the same
-  export PATH=$PATH:$(pwd)/node_modules/.bin
+  path_with_node_modules_binaries=$PATH:$(pwd)/node_modules/.bin
+  export PATH=$path_with_node_modules_binaries
 
-  local start_script=$(node -p "require('./package.json').scripts.start")
+  # Fetch 'start' script from package.json
+  start_script=$(node -p "require('./package.json').scripts.start")
 
+  # Execute the script
   $start_script
 }
 
 startup() {
-  if [ -z ${FIAAS_ENVIRONMENT} ]; then
-    log "FIAAS_ENVIRONMENT is unset, not looking for secrets";
+  if [ -z "$FIAAS_ENVIRONMENT" ]; then
+    log "FIAAS_ENVIRONMENT is unset; not looking for secrets";
```

Comment:
heh

Reply:
that was a Martincheck violation, not shellcheck